bindings/python/test/curve_fitting/interpolator/test_interpolator.py

```diff
@@ -0,0 +1,63 @@
+# Apache License 2.0
+
+import pytest
+
+from ostk.mathematics.curve_fitting import Interpolator
+
+
+@pytest.fixture
+def interpolation_type() -> Interpolator.InterpolationType:
```

Comment:
Update accordingly for this all other cases of `Interpolator.InterpolationType` -> `Interpolator.Type`